app/errors.py

```diff
@@ -0,0 +1,20 @@
+class VaccineError (Exception):
+    def __init__(self, message : str = "VaccineError") -> None:
+        super().__init__(message)
+        self.message = message
+
+    def __str__(self) -> str:
+        return self.message
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
```

Comment:
Consider overriding the `__str__` method in `NotVaccinatedError` to provide a specific error message, e.g., `def __str__(self) -> str: return 'Visitor is not vaccinated.'`.